client/state/data-layer/wpcom/me/send-verification-email/test/index.js

```diff
@@ -15,48 +9,41 @@ import { http } from 'state/data-layer/wpcom-http/actions';
 
 describe( 'send-email-verification', () => {
 	describe( '#requestEmailVerification', () => {
-		const dispatchSpy = spy();
 		const dummyAction = { type: 'DUMMY' };
 
-		requestEmailVerification( { dispatch: dispatchSpy }, dummyAction );
-
-		test( 'should dispatch HTTP request to plans endpoint', () => {
-			expect( dispatchSpy ).to.have.been.calledOnce;
-			expect( dispatchSpy ).to.have.been.calledWith(
-				http(
-					{
-						apiVersion: '1.1',
-						method: 'POST',
-						path: '/me/send-verification-email',
-					},
-					dummyAction
+		test( 'should dispatch HTTP request to me/send-verification-email endpoint', () => {
+			expect( requestEmailVerification( dummyAction ) ).toEqual(
+				expect.objectContaining(
+					http(
+						{
+							apiVersion: '1.1',
+							method: 'POST',
+							path: '/me/send-verification-email',
+						},
+						dummyAction
+					)
```

Comment:
Note that the unit tests for the handlers became so trivial that they are almost useless:
```js
function create() {
  return something;
}
expect( create() ).toEqual( something );
```
We are just duplicating `something` at two places. Can we test the handler is some better, more thorough way? Does it need any unit tests at all?